### PR TITLE
Fix Python-bound function signature (torch._C.Graph.addInput)

### PR DESCRIPTION
**Pull request by wschin**
In pytorch/torch/_C/__init__.pyi, Graph.addInput has signature
```python
  def addInput(self, name: str) -> Value: ...
```
which doesn't match the corresponding function
```cpp
  Value* addInput(const std::string& name = "") {
    return block_->addInput(name);
  }

```

in python_ir.cpp. This PR aligns the bound function on both C++ and Python sides. Without this PR, mypy will compain whenever a change contains some calls to `addInput`; for example,
image

### CODE REVIEW

File: test/jit/test_python_bindings.py [hunk 1]
```diff
@@ -84,6 +84,11 @@ def test_graph_create(self):
         with self.assertRaises(ValueError):
             gr.create("prim::Constant", [None])
 
+    def test_add_input(self):
+        gr = torch._C.Graph()
+        foo_value = gr.addInput("foo")
```
Comment (wschin) [inline test/jit/test_python_bindings.py:89]: This fails without this PR.

File: torch/csrc/jit/python/python_ir.cpp [hunk 2]
```diff
@@ -382,7 +382,11 @@ void initPythonIRBindings(PyObject* module_) {
           "Find all nodes",
           py::arg("kind"),
           py::arg("recurse") = true)
-      .def("addInput", [](Graph& g) { return g.addInput(); })
+      .def(
+          "addInput",
+          [](Graph& g, const std::string& name) { return g.addInput(name); },
+          "Add input to graph with optional name seed",
+          py::arg("name") = "")
```
Comment (wschin) [inline torch/csrc/jit/python/python_ir.cpp:389]: Use `""` so that `g.addInput()` still works as before.

**Review by wconstab — COMMENTED** [review]
This looks ok to me but i'd like one of the JIT team to verify too. @Krovatkin

**Review by davidberard98 — APPROVED** [review]
LGTM!

**Pull request closed without merging**